source/MaterialXGenOgsXml/GlslFragmentGenerator.cpp

```diff
@@ -36,6 +36,16 @@ string GlslFragmentSyntax::getVariableName(const string& name, const TypeDesc* t
 const string GlslFragmentGenerator::TARGET = "ogsxml";
 const string GlslFragmentGenerator::MATRIX3_TO_MATRIX4_POSTFIX = "4";
 
+// Streams of the isRequirementOnly type require special handling.
+static const StringSet OGS_IS_REQUIREMENT_ONLY =
```

Comment:
This duplicates https://github.com/autodesk-forks/MaterialX/blob/0ab83d01893fb6cfdfa670a0f1cb895130db72ba/source/MaterialXGenOgsXml/OgsXmlGenerator.cpp#L90 but it's probably better not to support `isRequirementOnly` at all (I'll elaborate below).